content/concepts/refs.mdx

```diff
@@ -2,241 +2,123 @@ import { Aside } from "~/components/configurable/Aside";
+This can be done by using [`onMount`](/reference/onMount) to wait until the element is attached to the DOM before accessing it:
 
 ```tsx
-let myDiv;
+import { onMount } from 'solid-js';
 
-<div ref={myDiv}>My Element</div>
-```
-
-<Aside>
-
-In TypeScript, you must use a definitive assignment assertion. 
-This signals to TypeScript that `myDiv` will definitely be assigned, even if TypeScript can not confirm it.
-This is particularly useful in Solid since the framework takes care of assigning the variable when the component is rendered.
-
-```tsx
-  let myDiv!: HTMLDivElement;
-```
-</Aside>
-
-
-## Why use refs?
-
-Refs are the most practical way to access DOM elements in Solid. They are localized, more 
-predictable, and easier to follow than the alternatives.
-
-
-### Why not use DOM selectors?
-
-In vanilla JavaScript, you would most likely use an element selector like `document.getElementById` 
-or `document.querySelector` to get a reference to a DOM element. Typically you know the element 
-exists in the DOM, so you can safely use these selectors. 
-
-However, in Solid, components are often added and removed from the DOM, so you can't always be 
-sure that the element exists. You might wonder why not just put a selector in the `onMount` so you
-can be sure the element exists like this.
-
-```tsx
 function Component() {
   onMount(() => {
     const element = document.getElementById('my-element');
-    // Do something with element
+    // Do something with the element...
   });
 
-  return <div id="my-element">My Element</div>;
+  return (
+    <div id="my-element">My Element</div>
+  )
 }
 ```
 
-Although this works, now this component can only be used once in the DOM. If you try to use it
-multiple times the selector will always select the first element. Using DOM selectors which are 
-global in the localized scope of a component is not a good idea.
-
+While this works, this component is now only able to be used once in the DOM.
+If you were to use this component multiple times, you would have multiple elements with the same `id`, which is invalid HTML.
 
-### Why not use JSX as a value?
+## JSX as a value
 
-You can use JSX as a value and assign it to a variable, which is similar to using 
-`document.createElement` in vanilla JavaScript. 
+When wanting to directly access the DOM element, you can use JSX as a value and assign it to a variable.
 
 ```jsx
 function Component() {
-  const element = <p>My Element</p>;
-  // Do something with element
+  const myElement = <div>My Element</div>
 
   return (
     <div>
-      {element}
+      {myElement}
     </div>
-  );
+  )
 }
 ```
 
-This isn't ideal because it requires you to separate the reference element and all of its children 
-from rest of the JSX tree. This can make reading through and understanding the component more 
-difficult. 
-
-## Callbacks
```

Comment:
@LadyBluenotes I'm not sure I understand why this was removed entirely, I think it is still good to know that you can use a callback

Reply:
It's a more relevant section to add to the refs reference page, I think. It's a bit disjointed to add into this for someone who just wants to know how to reference certain DOM elements